test/fuzz_group.cpp

```diff
@@ -543,22 +660,25 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
                             }
                         }
                         else if (type == type_Timestamp) {
-                            int64_t seconds = get_int64(s);
-                            int32_t nanoseconds = get_int32(s) % 1000000000;
-                            // Make sure the values form a sensible Timestamp
-                            const bool both_non_negative = seconds >= 0 && nanoseconds >= 0;
-                            const bool both_non_positive = seconds <= 0 && nanoseconds <= 0;
-                            const bool correct_timestamp = both_non_negative || both_non_positive;
-                            if (!correct_timestamp) {
-                                nanoseconds = -nanoseconds;
-                            }
-                            Timestamp value{seconds, nanoseconds};
+                            std::pair<int64_t, int32_t> values = get_timestamp_values(s);
+                            Timestamp value{values.first, values.second};
                             if (log) {
                                 *log << "g.get_table(" << table_ndx << ")->set_timestamp(" << col_ndx << ", "
                                      << row_ndx << ", " << value << ");\n";
                             }
                             t->set_timestamp(col_ndx, row_ndx, value);
                         }
+                        else if (type == type_Mixed) {
+                            if (log) {
+                                *log << "{\n";
+                            }
+                            Mixed mixed = construct_mixed(s, log);
+                            if (log) {
+                                *log << "\ng.get_table(" << table_ndx << ")->set_mixed(" << col_ndx << ", "
```

Comment:
Okay, I see it was cleverly put here. I think, for clarity, it would be better to keep at the end of each line in the construction of the mixed value.